keyboards/handwired/stef9998/split_5x7/rev1/rev1.h

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2022 stef9998
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#pragma once
+
+#include "split_5x7.h"
+
+#include "quantum.h"
+
+#ifdef USE_I2C
+#include <stddef.h>
+#ifdef __AVR__
+  #include <avr/io.h>
+  #include <avr/interrupt.h>
+#endif
+#endif
```

Comment:
```suggestion
```

Reply:
Just for my interest.
What is the reason all of that can be excluded? Is it somehow included automatically by defining I want to use I2C in my config?
And I was just using it some time ago and now it's not needed anymore?

Reply:
This is cruft left over from the old, old style of split keyboards, before it was migrated into the core 

These files are included in the files that are needed (such as the serial.c files).  

Basically, I can tell that this was copy and pasted from another board, since we've been actively removing these.

Reply:
Yes I copied the code from the Redox Keyboard at least a year ago or something. The old code there is from 2018, so that explains it :)
I'm going to just commit your suggestion on this one. The rest of your suggestions I also fixed to my abilities through commits.